labs/lab5/jump-to-here-playbook.yml

```diff
@@ -0,0 +1,38 @@
+--- 
+- hosts: localhost
+  become_method: sudo
+  tasks:
+  - name: build registry
+    command: podman build -t registry /home/ec2-user/summit-2018-container-lab/labs/lab1/registry 
+    become: true
+
+  - name: build mariadb
+    command: > 
+      podman build 
+      -t mariadb 
+      -f /home/ec2-user/summit-2018-container-lab/labs/lab3/mariadb/Dockerfile.reference /home/ec2-user/summit-2018-container-lab/labs/lab3/mariadb/
+    become: true
+
+  - name: build wordpress
+    command: >
+      podman build 
+      -t wordpress 
+      -f /home/ec2-user/summit-2018-container-lab/labs/lab3/wordpress/Dockerfile.reference
+      /home/ec2-user/summit-2018-container-lab/labs/lab3/wordpress/
+    become: true
+
+  - name: launch registry
+    command: podman run --name registry -p 5000 -d registry
```

Comment:
Once https://github.com/dustymabe/summit-2018-container-lab/pull/36 is merged, we have the students use a volume to persist the pushed images... not critical though.

Reply:
disregard ... we've moved away from persistence for registry